smalltalksrc/Slang/Gnuifier.class.st

```diff
@@ -144,10 +144,10 @@ Gnuifier >> gnuifyFrom: inputFile to: outFileStream [
 							ifFalse: [ 
 								inInterpret
 									ifTrue: [ "working inside interpret(); translate the switch statement"
-										(inLine beginsWith: '		case ') ifTrue: [ 
+										(inLine beginsWith: '				case ') ifTrue: [ 
 											| tokens |
 											tokens := inLine findTokens: '	 :'.
-											outLine := '		CASE(' , tokens second , ')'.
+											outLine := '				CASE(' , tokens second , ')'.
```

Comment:
this is work in progress, right? Would be nice to put it in a separate branch next time to avoid polluting the rest of super cool changes :)

Reply:
Understood :)